proparse/src/main/java/org/prorefactor/treeparser/TreeParserSymbolScope.java

```diff
@@ -48,8 +56,8 @@
   protected Map<Integer, Map<String, Symbol>> typeMap = new HashMap<>();
   protected Map<String, Variable> variableMap = new HashMap<>();
 
-  protected SymbolScope parentScope;
-  protected SymbolScopeRoot rootScope;
+  protected TreeParserSymbolScope parentScope;
+  protected TreeParserRootSymbolScope rootScope;
 
   // Initialization block
   {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the contents of this initializer to a standard constructor or to field initializers. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1171)